skbio/stats/ordination/_principal_coordinate_analysis.py

```diff
@@ -24,7 +25,94 @@
 #   so, so I'm not doing that.
 
 
-class PCoA(Ordination):
+def _eigh(F_matrix, **ignored):
+    """Wrapper around np.linalg.eigh
```

Comment:
we need to support `**kwargs` for the methods, and in this method they are ignored